src/Microsoft.DotNet.Cli.Utils/CommandResolver.cs

```diff
@@ -181,8 +181,7 @@ private static CommandSpec ConfigureCommandFromPackage(string commandName, IEnum
 
                 if (depsPath != null)
                 {
-                    additionalArgs.Add("--depsfile");
-                    additionalArgs.Add(depsPath);
+                    additionalArgs.Add($"--depsfile:{depsPath}");
```

Comment:
Have a few questions:
Is this breaking now?
Do we want this inside one element of argv?
Why is this a special case?